source/platforms/kubeflow.md

```diff
@@ -1,2 +1,233 @@
+---
+alt: Screenshot of Jupyter Lab with the “10 minutes to cuDF and dask-cuDF” notebook open
+---
+```
+
+## Scaling out to many GPUs
+
+Many of the RAPIDS libraries also allow you to scale out your computations onto many GPUs spread over many nodes for additional acceleration. To do this we leverage [Dask](https://www.dask.org/), an open source Python library for distributed computing.
+
+To use Dask, we need to create a scheduler and some workers that will perform our calculations. These workers will also need GPUs and the same Python environment as your notebook session. Dask has [an operator for Kubernetes](/tools/kubernetes/dask-operator) that you can use to manage Dask clusters on your KubeFlow cluster.
+
+### Installing the Dask Kubernetes operator
+
+To install the operator we need to create any custom resources and the operator itself, please [refer to the documentation](https://kubernetes.dask.org/en/latest/operator_installation.html) to find up-to-date installation instructions. From the terminal run the following commands.
+
+```console
+$ kubectl apply -f https://raw.githubusercontent.com/dask/dask-kubernetes/main/dask_kubernetes/operator/deployment/manifests/daskcluster.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/dask/dask-kubernetes/main/dask_kubernetes/operator/deployment/manifests/daskworkergroup.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/dask/dask-kubernetes/main/dask_kubernetes/operator/deployment/manifests/daskjob.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/dask/dask-kubernetes/main/dask_kubernetes/operator/deployment/manifests/operator.yaml
+```
+
+Verify our resources were applied successfully by listing our Dask clusters. Don’t expect to see any resources yet but the command should succeed.
+
+```console
+$ kubectl get daskclusters
+No resources found in default namespace.
+```
+
+You can also check the operator pod is running and ready to launch new Dask clusters.
+
+```console
+$ kubectl get pods -A -l application=dask-kubernetes-operator
+NAMESPACE       NAME                                        READY   STATUS    RESTARTS   AGE
+dask-operator   dask-kubernetes-operator-775b8bbbd5-zdrf7   1/1     Running   0          74s
+```
+
+Lastly, ensure that your notebook session can create and manage Dask custom resources. To do this you need to edit the `kubeflow-kubernetes-edit` cluster role that gets applied to notebook pods. Add a new rule to the rules section for this role to allow everything in the `kubernetes.dask.org` API group.
+
+```console
+$ kubectl edit clusterrole kubeflow-kubernetes-edit
+…
+rules:
+…
+- apiGroups:
+  - "kubernetes.dask.org"
+  verbs:
+  - "*"
+  resources:
+  - "*"
+…
+```
+
+### Creating a Dask cluster
+
+Now you can create `DaskCluster` resources in Kubernetes that will launch all the necessary pods and services for our cluster to work. This can be done in YAML via the Kubernetes API or using the Python API from a notebook session.
```

Comment:
```suggestion
Now you can create `DaskCluster` resources in Kubernetes that will launch all the necessary pods and services for our cluster to work. This can be done in YAML via the Kubernetes API or using the Python API from a notebook session as shown in this section.
```